{"uuid": "09540944-6a08-4d38-8403-633a0dcb6693", "children": ["377a5c91-c189-4f7e-a32f-386f8d4ee0b6"], "befores": [{"name": "log_on_failure", "status": "passed", "start": 1747732331728, "stop": 1747732331728}], "afters": [{"name": "log_on_failure::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: disconnected: not connected to DevTools\n  (failed to check if window was closed: disconnected: not connected to DevTools)\n  (Session info: MicrosoftEdge=117.0.2045.35)\nStacktrace:\n0   msedgedriver                        0x0000000104fb3a10 msedgedriver + 4160016\n1   msedgedriver                        0x0000000104fabad0 msedgedriver + 4127440\n2   msedgedriver                        0x0000000104bf5338 msedgedriver + 234296\n3   msedgedriver                        0x0000000104be4d4c msedgedriver + 167244\n4   msedgedriver                        0x0000000104be4a64 msedgedriver + 166500\n5   msedgedriver                        0x0000000104c59428 msedgedriver + 644136\n6   msedgedriver                        0x0000000104c24d04 msedgedriver + 429316\n7   msedgedriver                        0x0000000104c25bb8 msedgedriver + 433080\n8   msedgedriver                        0x0000000104f82070 msedgedriver + 3956848\n9   msedgedriver                        0x0000000104f856cc msedgedriver + 3970764\n10  msedgedriver                        0x0000000104f851bc msedgedriver + 3969468\n11  msedgedriver                        0x0000000104f8a500 msedgedriver + 3990784\n12  msedgedriver                        0x0000000104f85d7c msedgedriver + 3972476\n13  msedgedriver                        0x0000000104f6861c msedgedriver + 3851804\n14  msedgedriver                        0x0000000104f9d47c msedgedriver + 4068476\n15  msedgedriver                        0x0000000104f9d5a8 msedgedriver + 4068776\n16  msedgedriver                        0x0000000104faad5c msedgedriver + 4123996\n17  libsystem_pthread.dylib             0x000000018b0fef94 _pthread_start + 136\n18  libsystem_pthread.dylib             0x000000018b0f9d34 thread_start + 8\n\n", "trace": "  File \"/Users/lekhraj/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lekhraj/.venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 906, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/lekhraj/StudentAndroidApp/Student-App-Web/Test/conftest.py\", line 70, in log_on_failure\n    allure.attach(driver.get_screenshot_as_png(), name=\"Screenshot\", attachment_type=AttachmentType.PNG)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lekhraj/.venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 839, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lekhraj/.venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 850, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lekhraj/.venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/lekhraj/.venv/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1747732348804, "stop": 1747732348809}], "start": 1747732331728, "stop": 1747732348809}